YoutubeExplode/YoutubeClient.cs

```diff
@@ -404,10 +410,17 @@ public YoutubeClient()
 
                     // Fix for Hebrew
                     if (lang == "iw") lang = "he";
-
-                    var culture = new CultureInfo(lang);
-                    var caption = new ClosedCaptionTrackInfo(url, culture, isAuto);
-                    captions.Add(caption);
+                    // Fix for Yi
+                    if (lang == "yi") lang = "ii";
+
+                    try {
```

Comment:
Does it actually throw an exception? In my case it just returns an "Unknown Language (iso_code)" object.